.travis.yml

```diff
@@ -5,9 +5,9 @@ script: bash -ex .travis-ci.sh
 env:
   global:
   - UPDATE_GCC_BINUTILS=1
-  - EXTRA_REMOTES="https://github.com/yomimono/mirage-dev.git#remove-console"
+  - EXTRA_REMOTES="https://github.com/mirage/mirage-dev.git"
```

Comment:
this charrua-core pin also looks fishy... hasn't that now be merged to master (and shouldn't we have a charrua-core package in mirage-dev)? (sorry for nitpicking, I'll go back and code)